src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java

```diff
@@ -98,8 +98,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
         TFM_BanManager.addUuidBan(new TFM_Ban(TFM_UuidManager.getUniqueId(player), player.getName(), sender.getName(), null, reason));
 
         // kick Player:
-        player.kickPlayer(ChatColor.RED + "GTFO" + (reason != null ? ("\nReason: " + ChatColor.YELLOW + reason) : ""));
-
+        player.kickPlayer(ChatColor.RED + "You have been banned, GTFO!" + (reason != null ? ("\nReason: " + ChatColor.YELLOW + reason) : "") + " - " + sender.getName());
```

Comment:
I prefer the first line to stay just "GTFO".

Please state the ban author on a new line, much like the reason.